### ISSUE TITLE
Review recent docs PRs and cross-link where one concept builds on/relates to another

### ISSUE THREAD

**Issue by anthrocypher** [sourcecred/sourcecred#1050]
Recent documentation efforts have made great strides in uncovering how SourceCred works for would-be contributors. Going one step further and adding some cross-links to associated in-line docs elsewhere in the repo would create the equivalent of signposts so folks know where they are on the map, relative to other destinations.

**Comment by decentralion** [comment]
Yep, this is a good point. I'll keep an eye out for opportunities to create cross references, and I encourage you to PR any suggestions you have!

**Comment by wpank** [comment]
@anthrocypher to clarify, do you mean adding cross-links to in-line docs elsewhere in in other in-line docs? ie `file 1` has a comment that points to a comment in `file 2`. Or do you mean add cross-links to external docs?
